content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+---
+title: Jenkins Configuration as Code (JCasC)
+layout: section
+wip: true
+---
+
+The Jenkins Configuration as Code (JCasC) feature defines Jenkins configuration parameters
+in a human-readable YAML file that can be stored under source code.
+This essentially captures the configuration parameters and values that are used when configuring Jenkins from the web UI.
+The configuration can then be modified by editing this file and then applying it.
+
+Configuring and tuning Jenkins and its plugins is a complex process,
+with dozens of parameters to set within the web UI `manage` section.
+Traditionally, experienced Jenkins administrators
```

Comment:
JCasC has been around for years at this point, not sure if this is required anymore, you could possibly drop this paragraph

Reply:
Project started in 2017.
v1 release in 2019: https://github.com/jenkinsci/configuration-as-code-plugin/releases/tag/configuration-as-code-1.0

Reply:
I didn't like that first sentence but was trying to be respectful of what is in the current docs...  I've deleted the first sentence and turned the info about using Groovy .init scripts into a NOTE.  It seems worth mentioning for someone who might wonder if they can still do that.